KIF Tests/GestureTests.m

```diff
@@ -193,6 +193,18 @@ - (void)testSwipingDownWithIdentifier
     [tester waitForViewWithAccessibilityLabel:@"Down"];
 }
 
+- (void)testSwipeFromScreenEdgeLeft
```

Comment:
Worth having a similar test for verifying that dragFromPoint doesn't trigger? Or is it reasonable to treat that as undefined behavior and be untested?